serialization.go

```diff
@@ -76,20 +76,22 @@ func SerializeSTHSignatureInput(sth SignedTreeHead) ([]byte, error) {
 }
 
 // CreateX509MerkleTreeLeaf generates a MerkleTreeLeaf for an X509 cert
-func CreateX509MerkleTreeLeaf(cert ASN1Cert, timestamp uint64) *MerkleTreeLeaf {
+func CreateX509MerkleTreeLeaf(cert ASN1Cert, timestamp uint64, extensions CTExtensions) *MerkleTreeLeaf {
```

Comment:
These methods are exported, so changing their signature would require a version update and might break deps outside of this repo. Let's create new wrapper functions called something along the lines of `CreateX509MerkleTreeLeafExt` to avoid this? Then we can update the tools, leave the CTFE code as it is, and it won't break downstream dependencies.

Reply:
Sure. I hesitated to change the signatures to break the external deps, but I was also afraid to added more functions to complicate the package/usage.  Now that you suggested the latter, I'll update it.